packages/firestore/test/unit/local/indexeddb_schema.test.ts

```diff
@@ -29,8 +29,15 @@ import {
 } from '../../../src/local/indexeddb_schema';
 import { SimpleDb, SimpleDbTransaction } from '../../../src/local/simple_db';
 import { PersistencePromise } from '../../../src/local/persistence_promise';
+import { ClientId } from '../../../src/local/shared_client_state';
```

Comment:
You have a TODO in your PR description about renaming this file, but that isn't greppable.  Can you add a TODO(multitab) in this file if you really intend to rename this?

Reply:
Added